Assets/Mirror/Runtime/Transport/Kcp/KcpTransport.cs

```diff
@@ -14,6 +14,7 @@ public class KcpTransport : Transport
         [Header("Transport Configuration")]
         public ushort Port = 7777;
 
+        public KcpDelayMode delayMode = KcpDelayMode.Fast3;
```

Comment:
Maybe default should be Normal?

Reply:
It should be but all of the testing we have done so far was on fast3. So I didn't want to disrupt anything out of the blue with this.

Reply:
it may be causing the tests to run slower,  not sure what is going on, but the tests are taking for ever